.changeset/angry-ligers-judge.md

```diff
@@ -0,0 +1,17 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/datastore": minor
+"@fluidframework/garbage-collector": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/runtime-utils": minor
+"@fluidframework/test-end-to-end-tests": minor
```

Comment:
@agarwal-navin Since there were two notes in breaking.md I made two changesets. I marked all the packages edited in this commit for both changesets, but if some packages should be excluded for one or the other we can do that.